atl-checker/src/main.rs

```diff
@@ -160,6 +159,42 @@ fn main_inner() -> Result<(), String> {
                 },
             )?
         }
+        ("analyse", Some(analyse_args)) => {
+            let input_model_path = analyse_args.value_of("input_model").unwrap();
+            let model_type = get_model_type_from_args(&analyse_args)?;
+            let formula_path = analyse_args.value_of("formula").unwrap();
+            let formula_format = get_formula_format_from_args(&analyse_args)?;
+
+            fn analyse_model<G>(graph: ATLDependencyGraph<G>, v0: ATLVertex)
+            where
+                G: GameStructure + Send + Sync + Clone + Debug + 'static,
+            {
+                println!("Analysing EDG ...");
+                let mss = minimum_solve_set(&graph, v0);
+                for (vertex, assignment) in mss {
+                    println!("{}: {}", vertex, assignment.len());
+                }
+                println!("Done");
```

Comment:
```suggestion
```

Looks like testprinting, if program exits without panic and with retcode 0, it's done, so no need to dirty stdout.